.gitmodules

```diff
@@ -40,7 +40,7 @@
 [submodule "external/eigen"]
 	path = external/eigen
 	url = https://gitlab.com/libeigen/eigen.git
-	branch = 3.3
+	branch = 3.4
```

Comment:
Why?

Reply:
for upgrade external dependencies to latest versions

Reply:
> for upgrade external dependencies to latest versions

I mean, why upgrading dependencies? Are they necessary to the ARM version?